contentcuration/contentcuration/frontend/shared/views/StudioChip.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+
+  <div
+    ref="chip"
+    class="studio-chip"
+    :class="{
+      'studio-chip--small': small,
+      'studio-chip--clickable': close || $listeners.click,
+      'studio-chip--active': isActive,
+    }"
+    :style="chipStyles"
+    @click="handleClick"
+  >
+    <div class="studio-chip__content">
+      <div class="studio-chip__text">
+        <slot>
+          {{ text }}
+        </slot>
+      </div>
+
+      <KIconButton
```

Comment:
This icon button needs an accessible name attached to it, like `ariaLabel ="Remove {username}"`, using the [KIconButton ariaLabel prop](https://design-system.learningequality.org/kiconbutton#prop:ariaLabel).